packages/components/collapsible-panel/src/header-icon.tsx

```diff
@@ -1,19 +1,34 @@
 import { css } from '@emotion/react';
-import { designTokens } from '@commercetools-uikit/design-system';
+import { designTokens, useTheme } from '@commercetools-uikit/design-system';
 import { AngleDownIcon, AngleRightIcon } from '@commercetools-uikit/icons';
 
 const sizeIconContainer = '22px';
 const sizeIconContainerSmall = '14px';
 
-const getArrowColor = ({
-  tone,
-  isDisabled,
-}: Pick<THeaderIcon, 'isDisabled' | 'tone'>) => {
+const getArrowColor = (
+  { tone, isDisabled }: Pick<THeaderIcon, 'isDisabled' | 'tone'>,
+  theme: 'default' | 'test'
+) => {
   if (isDisabled) return 'neutral60';
-  if (tone === 'urgent') return 'surface';
+  if (tone === 'urgent') {
+    if (theme === 'default') {
+      return 'surface';
+    } else {
+      return 'warning';
+    }
+  }
+
   return 'solid';
 };
 
+const checkIfTestOrDefault = (
+  theme?: 'default' | 'test',
```

Comment:
I don't think the name `test` will stay for long-term, so I wouldn't couple our names with it. (same in other places)

Reply:
There's a `ThemeName` type exported from the `design-system` package you could use here.

I wouldn't worry too much about the naming of the test as we will need to just remove all this transitioning code after the new theme is finally released.